src/hotspot/share/opto/type.cpp

```diff
@@ -1113,6 +1113,72 @@ void Type::dump_stats() {
 }
 #endif
 
+//------------------------------category---------------------------------------
+#ifndef PRODUCT
+Type::CATEGORY Type::category() const {
+  const TypeTuple* tuple;
+  switch (base()) {
```

Comment:
Might be more readable if switch cases are indented.

Reply:
Done.